### PR TITLE
Fixing module function to use correct rabbitmq command

### PR DESCRIPTION
**Pull request by teoyaomiqui**
#41458

### What does this PR do?
Fixes rabbitmq module function to form correct command

### What issues does this PR fix or reference?
41458
Related to issue https://github.com/saltstack/salt/issues/41458

### Previous Behavior
rabbitmq_user.present fails with rabbitmq-servers versions below 3.5.6

### New Behavior
rabbitmq_user.present succeeds with rabbitmq-servers versions below 3.5.6

### Tests written?

No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

### CODE REVIEW

**Review by denyskulykov — APPROVED** [review]

**Review by izadorozhna — APPROVED** [review]

**Comment by akissa** [comment]
This change breaks backwards compartibility

**Comment by teoyaomiqui** [comment]
I don't have much experience with this module, so could you please point out how exactly and in what conditions backwards compatibility would be broken?

Thanks in advance!

File: salt/modules/rabbitmq.py [hunk 1]
```diff
@@ -473,7 +473,7 @@ def check_password(name, password, runas=None):
             return False
         return True
 
-    cmd = ('rabbit_auth_backend_internal:check_user_login'
+    cmd = ('rabbit_auth_backend_internal:user_login_authentication'
```
Comment (epcim) [inline salt/modules/rabbitmq.py]: do the same thing what puppet folks: https://github.com/puppetlabs/puppetlabs-rabbitmq/pull/322/files#diff-c62722635df69c579feeb9bf9534b82dR48

"version" is available in the code, so it's not a problem to keep backward compatibility.

**Review by epcim — CHANGES_REQUESTED** [review]

**Comment by teoyaomiqui** [comment]
There another function in rabbitmq, as far as i've checked,  it is available in all versions. I've changed the commit, please take a look at it.

**Comment by epcim** [comment]
best if you could tes the change on lab/staging.On 26 May 2017 21:52, Kostiantyn Kalynovskyi <notifications@github.com> wrote:There another function in rabbitmq, as far as i've checked,  it is available in all versions. I've changed the commit, please take a look at it.

**Comment by teoyaomiqui** [comment]
I did check it on two envs.

**Comment by cachedout** [comment]
@epcim and @akissa How does this look to you now? Are there additional changes you would like to see?

**Comment by cachedout** [comment]
@epcim and @akissa. Last call for feedback. :]

**Comment by akissa** [comment]
If i can remember correctly the version of rabbitmq shipped in centos 6 does not have the function. I am so busy with other stuff i do not have time confirm that, @teoyaomiqui if you can please confirm that.

**Comment by cachedout** [comment]
Thanks, @akissa 

@teoyaomiqui Did you see the above comment?

**Comment by teoyaomiqui** [comment]
@cachedout i'll check it out next week, I've been quite busy during two past weeks, my apologies.

**Comment by cachedout** [comment]
@teoyaomiqui Sounds good. Many thanks.

**Comment by cachedout** [comment]
@teoyaomiqui Were you able to look at the above comment?

**Comment by cachedout** [comment]
@teoyaomiqui Because we haven't heard back from you, I need to close this PR for the time being. When you are ready to come back to it, please let me know and we'll happily re-open this. Thanks.

**Pull request closed without merging**